src/common/utils.ts

```diff
@@ -20,6 +20,9 @@
 import { Runtime } from '../index'
 
 
+export const nullAddress = '0x0000000000000000000000000000000000000000';
```

Comment:
shall we use caps-lock style for definition constants? NULL_ADDRESS and NULL_BYTES_32 ?

Reply:
Usual code style does not use upper case constants. We had a JavaScript code style, that forbade this. We carried over this convention when moving to TypeScript.